source/math/FloatVectorList.ooc

```diff
@@ -432,9 +433,51 @@ FloatVectorList: class extends VectorList<Float> {
 		}
 		(crossCorrelation, offsets)
 	}
+	// Assumes equi-distant samples
+	// https://en.wikipedia.org/wiki/Spline_interpolation
+	interpolateSpline: func (numberOfPoints: Int) -> This {
+		result := This new(this count + numberOfPoints * (this count - 1))
+		thisPointer := this pointer as Float*
+		coefficientMatrix := This _generateCubicSplineCoefficients(this count)
+		b := FloatMatrix new(1, this count) take()
```

Comment:
I discussed this with @niklasborwell before and we agreed that `b` is a sufficient name here and it is also what we use in some other places (which I can not talk about here). I can change it if you want though.